### PR TITLE
Update default pools

### PR DESCRIPTION
**Pull request by fireice-uk**
@psychocrypt as per our discussion I'm adding minemonero.pro along with a couple small pools that I think could use some extra traffic.

### CODE REVIEW

File: config.txt [hunk 1]
```diff
@@ -93,8 +93,15 @@ null,
...
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%
+ * pool.usxmrpool.com:3333  - 0.5%
+ * pool.xmr.pt:3333         - 0.4%
+ *
+ * We feature pools up to 1MH/s here. For a more complete list see M5M400's pool list at www.moneropools.com
  */
-"pool_address" : "pool.supportxmr.com:3333",
+"pool_address" : "pool.usxmrpool.com:3333",
```
Comment (psychocrypt) [inline config.txt]: This pool is showing the following message on the start page: 
```
We do not tolerate pool hoppers. If you are identified as one, you will be blocked
 from the pool, and any balance you may accumulate won't be paid. 
Pool hopping isn't fair for anyone, so we can't tolerate it in any form. 
If you aren't sure what pool hopping is, this message is most probably not for you.
```
It is absolutely unclear how this detection is working and I am not sure if some how is mining for a view hours than leave and came back later is identified as pool hopper. IMO a pool should be neutral and not evaluating the miner behave.
@fireice-uk can we remove this pool?

File: config.txt [hunk 2]
```diff
@@ -93,8 +93,15 @@ null,
  * pool_address	  - Pool address should be in the form "pool.supportxmr.com:3333". Only stratum pools are supported.
  * wallet_address - Your wallet, or pool login.
  * pool_password  - Can be empty in most cases or "x".
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%
```
Comment (psychocrypt) [inline config.txt]: This pool has currently no web front end. Have you ever seen the webpage online?

File: config.txt [hunk 3]
```diff
@@ -93,8 +93,15 @@ null,
  * pool_address	  - Pool address should be in the form "pool.supportxmr.com:3333". Only stratum pools are supported.
  * wallet_address - Your wallet, or pool login.
  * pool_password  - Can be empty in most cases or "x".
+ *
+ * Featured small pools       fee    
+ * pool.minemonero.pro:3333 - 0.9%
+ * pool.usxmrpool.com:3333  - 0.5%
+ * pool.xmr.pt:3333         - 0.4%
```
Comment (psychocrypt) [inline config.txt]: The website is not available.

File: config.txt (hunk 2 above)
Comment (fireice-uk) [inline config.txt]: https://minemonero.pro, it seems to be online

File: config.txt (hunk 3 above)
Comment (fireice-uk) [inline config.txt]: https://pool.xmr.pt - seems online now but no mining activity, we might need to re-evaluate that one

File: config.txt (hunk 1 above)
Comment (fireice-uk) [inline config.txt]: Pool hopping is a real problem - you can throw 1MH/s from nicehash for an hour and collect most of the reward from the pool when they eventually find a block. Of course the proper fix is technical (PPLNS), do you think this should disqualify them?

**Comment by fireice-uk** [comment]
In general feel free to add some more, especially instead of pool.xmr.pt I'm not sure what's going on there.

File: config.txt (hunk 2 above)
Comment (psychocrypt) [inline config.txt]: This pool has 2.2 MH/s this means the pool is over 1 MH/s.

File: config.txt (hunk 3 above)
Comment (psychocrypt) [inline config.txt]: The pool is offline again

**Comment by psychocrypt** [comment]
Mhh I am not sure if this PR is good for our low time budget. To find small pools those are stable and online is hard and the list must be checked very often. This would cost us a lot of time and support tickets if a pool is down.
IMO we should close this PR.

**Comment by fireice-uk** [comment]
I don't quite buy into the time argument - never seen a ticket here about a pool. And I don't particularly like the status quo of sending users towards a single pool. 

Another option is to blank it and only include list of the pools in the comment, but then we are reducing usability. If you would support that or have other ideas let's discuss them here.

**Comment by miziel** [comment]
Minemonero.pro just became quite big :)
Why not give the link to moneropools.com with suggestion to use smaller pools?

**Comment by fireice-uk** [comment]
@miziel Correct. I think both minemonero.pro and pool.xmr.pt are primarily devices to spend AWS credits of their respective owners =). With regard to linking to the pool list, that's the other option I outlined, however it means usability loss.

**Comment by fireice-uk** [comment]
@psychocrypt Can you approve the final version?

File: config.txt (hunk 1 above)
Comment (frost-monkey) [inline config.txt]: Hi! Pool owner here.

I'll make things clearer on the website soon, but we do not have automatic hopping detection. I do my best to block NiceHash's IPs. If I (we) catch someone sending, for example, 300 KH/s for 50 minutes than running away, I do my best to contact the person, and if I get no answer (or person doesn't want to talk), I block that miner's wallet and IP addresses. In two cases only, I have decided to not send the funds to these miners as they were CLEARLY abusing the pool.

And to make things very clear: there is absolutely no problem in connecting to the pool 10 or 20 hours a day. Many miners do so as it's simply a hobby for them (like gamers who stop mining while gaming). PPLNS is a "solution" but it penalizes smaller hobbyist miners, and I'm not sure I want to implement it.

PS: English is my second language. Please be indulgent. :-)

**Pull request merged**